wordpress-mobile/WordPress-Android
# Fix gemfile in 16.7 release branch

Fixes potential CI issues in `release/16.7`

To test:
- Ensure CI passes
- Once it does, run it again to verify that it also works fine when using the bundler cache

PR submission checklist:
